components/SEO.tsx

```diff
@@ -35,6 +42,12 @@ const CommonSEO = ({ title, description, ogType, ogImage, canonicalUrl }: Common
       ) : (
         <meta property="og:image" content={ogImage} key={ogImage} />
       )}
+
+      <meta name="twitter:card" content="summary" />
+      <meta name="twitter:site" content="@tailscale" />
+      <meta name="twitter:description" content={description} />
```

Comment:
Lets also add title.

```suggestion
      <meta name="twitter:title" content={title} />
      <meta name="twitter:description" content={description} />
```